frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -141,6 +142,17 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     combinedDisk.getStorageClassName() || '',
   );
 
+  React.useEffect(() => {
+    if (!isEditing) {
+      const defaultStorageClass = getLoadedData(storageClasses, []).find(
+        (sc) => getAnnotations(sc, {})[DEFAULT_SC_ANNOTATION] === 'true',
+      );
+
+      setStorageClassName(getName(defaultStorageClass) || '');
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isLoaded(storageClasses)]);
+
   const [size, setSize] = React.useState<string>(
     combinedDiskSize ? `${combinedDiskSize.value}` : '',
   );
```

Comment:
we should disable the storageClass field until storageClasses are loaded

Reply:
storageClass dropdown is now disabled until all sc are loaded